src/Tasks/FileIO/WriteLinesToFile.cs

```diff
@@ -51,11 +51,10 @@ public class WriteLinesToFile : TaskExtension, IIncrementalTask
         /// <summary>
         /// Question whether this task is incremental.
         /// </summary>
-        /// <remarks>When question is true, then this task would not write to disk.  If CanBeIncremental is true, then error out.</remarks>
+        /// <remarks>When question is true, then error out if WriteOnlyWhenDifferent would have
+        /// written to the file.</remarks>
         public bool FailIfNotIncremental { get; set; }
 
-        public bool CanBeIncremental => WriteOnlyWhenDifferent;
```

Comment:
APIScan is barking, is there a way to remove it.  This was an leftover from Rename.

Reply:
Added `[Obsolete]`